frontend/src/features/permits/pages/Refund/RefundPage.tsx

```diff
@@ -207,6 +213,23 @@ export const RefundPage = ({
     clearErrors("transactionId");
   };
 
+  /**
+   * Function to prevent non-numeric input as the user types
+   */
+  const filterNonNumericValue = (input?: string) => {
+    if (!input) return "";
+    // only allows 0-9 inputs
+    return input.replace(/[^\d]/g, "");
```

Comment:
Is this correct? you're just removing anything non-digit from the input. Shouldn't you display an error message asking the user to correct their input instead of just deleting non-numeric characters in their input?

Reply:
@krishnan-aot during todays dev sync Michael has confirmed that we are okay with this behavior - an error message pertaining to non-numeric values may be implemented in the future